app/javascript/dashboard/components/ui/AnnouncementPopup.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <div class="announcement-popup">
+    <span v-if="popupMessage" class="popup-content">
+      {{ popupMessage }}
+      <span class="route-url" @click="onClickOpenPath">
+        {{ routeText }}
```

Comment:
Can we combine `popupMessage` and `routeText` in to one prop and render using v-html?

Reply:
@muhsin-k I think @click will not work while using v-html.